### ISSUE TITLE
Confused about unknown localization

### ISSUE THREAD

**Issue by jinweiisgreat** [JosephKJ/OWOD#105]
Hi Joseph, thank you for your great work!
Here, I have a question about how to localize the unknown instance.
We know that unknown candidate proposals are from top-k region proposals which do not overlap the ground truth proposals, right?
But according to RPN loss function:
![image](https://user-images.githubusercontent.com/69202833/206357158-3c08883b-52bf-409d-954c-17baf0c5837d.png)
when $p_i^\*$ is 1, the region proposal participates in training. But for unknown objects, the $p_i^\*$ will be 0, the proposal of unknown objects cannot participate in training, right?
So, from my understanding, we just using anchors as proposals to localize unknown objects, right?
If I have misunderstood, please correct me. Thank you!
